xarray/core/groupby.py

```diff
@@ -21,6 +24,14 @@
 )
 from .variable import IndexVariable, Variable, as_variable
 
+XARRAY_NUMPY_GROUPIES = os.environ.get("XARRAY_NUMPY_GROUPIES", "False").lower() in (
+    "true",
+    "1",
+)
```

Comment:
I was thinking could we move this to our `options`, but I guess this is set too early for `options` to take effect.

Would people be OK having it as a default? Maybe we do a release with this sort of experimental flag and then we set it as the default?

Reply:
:+1: for options eventually.